pages/troubleshooting/restore-edison.md

```diff
@@ -0,0 +1,43 @@
+## Help I bricked my Edison
+
+So your Edison is unresponsive or for some bazaar reason you want to restore your Edison
+to its unresinified state.
+
+Its pretty easy to reflash the boring old Intel Yocto image onto the Edison. Just follow these steps.
+
+* Go to the [Intel Edison Downloads page][edison-dl-page] and download the latest version of Intel Edison® Board Firmware Software Release 2.1. At the time of writing it is [Release 2.1 Yocto* complete image][dl-link].
```

Comment:
Should probably only specify the version 2.1 once, ie `the latest version of Intel Edison® Board Firmware Software Release 2.1` -> `the latest version of Intel Edison® Board Firmware Software Release`